sdk/data/aztables/transactional_batch.go

```diff
@@ -346,7 +348,7 @@ func (t *Client) generateEntitySubset(transactionAction *TransactionAction, writ
 			entity[partitionKey].(string),
 			entity[rowKey].(string),
 			opts,
-			qo,
+			&generated.QueryOptions{},
```

Comment:
Why does `InsertMerge` need different `QueryOptions`?

Reply:
The `qo` passed an `ETag` of `*`, which returns a ResourceNotFoundError if the entity does not already exist. Passing the empty struct will not send that header at all which creates the entity if it does not exist, which is the desired behavior.